docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Manage your Applications using Kyma and Compass Console UI
+type: Tutorial
+---
+
+This tutorial presents an end-to-end use case scenario that shows how to connect an external Application to Compass so that its lambda calls and gets the order services. While going through this tutorial, you will navigate between two UI views:
+- Compass UI, where you create connections between Applications, Runtimes, and Scenarios
+- Kyma Console UI, where you manage resources used in your Application, such as services, lambdas, and bindings
+
+## Prerequisites
+
+Use the [HTTP DB Service](https://github.com/kyma-project/examples/tree/master/http-db-service) Application to go through this tutorial. Prepare the following:
+- [File](./assets/http-db-service-deployment.yaml) that contains the HTTP DB Service definition and deployment
+- HTTP DB Service [API spec](./assets/http-db-api-spec.yaml)
+- [`call-order-service`](./assets/lambda.yaml) lambda function
+- Kyma cluster with the Compass module enabled
+
+>**NOTE:** Read [this](#installation-enable-compass-in-kyma-compass-as-a-central-management-plane) document to learn how to install Kyma with the Compass module enabled.
+
+## Steps
+
+### Connect the external Application
+
+1. Log in to the Kyma Console and create the `test` Namespace by clicking the **Add new namespace** button.
+2. In the **Overview** tab in your Namespace, click the **Deploy new resource** button and select the file with HTTP DB Service and deployment to connect the Application.
```

Comment:
and select the file with HTTP DB Service and deployment to connect the Application. -> and use the file with HTTP DB Service. (maybe add link again to file)